paparazzi/src/test/java/app/cash/paparazzi/internal/apng/ApngReaderTest.kt

```diff
@@ -37,7 +37,13 @@ class ApngReaderTest {
       val expectedFile = javaClass.classLoader.getResource("simple_animation_$i.png")
       val expectedImage = ImageIO.read(expectedFile)
       val actualImage = reader.readNextFrame()!!
-      ImageUtils.assertImageSimilar(expectedFile!!.path, expectedImage, actualImage, 0.0)
+      ImageUtils.assertImageSimilar(
+        expectedFile!!.path,
+        File(file.path).parentFile,
```

Comment:
if the test fails, will this directory even be writable?  if so, will it be cleaned up?

let's pass a temporary folder instead?